### DISCUSSION TITLE
Explicit MSGV

### DISCUSSION THREAD

**Discussion by vurvdev** [wiremod/wire#2921] [Suggestions]
Want an explicit "Minimum supported gmod version", that will be on the README file and if a server loads with a wiremod version newer than this, it'll give off an `ErrorNoHalt`.

Fixes and compatibility with these version differences won't be guaranteed.

**Comment by marchc1** [comment]
I think that it should print something in chat alongside the ErrorNoHalt when the user loads in, would make it a bit more obvious to users + owners IMO
